dev-docs/publisher-api-reference/registerSignalSources.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: api_prebidjs
+title: pbjs.registerSignalSources()
+description: enabling ESP (encrypt signals) for Open Bidding
+---
+
+{: .alert.alert-info :}
+To use this function, include the [UserId module](/dev-docs/modules/userId.html) in your Prebid.js build.
+
+This function will register all configured encrypted signals as described in the [UserId module ESP configuration](/dev-docs/modules/userId.html#esp-configurations). 
```

Comment:
It might make sense to list which known signals google will take up

Gam dialogue lists these; one's that are not id providers are not clear what they send

AdColony
check_box_outline_blank
Adform ID Fusion
check_box_outline_blank
AppLovin
check_box
Criteo
check_box_outline_blank
Facebook
check_box
ID5
check_box
Index Exchange
check_box_outline_blank
LiveRamp Authenticated Traffic Solution (ATS)
check_box_outline_blank
Lotame Panorama ID
check_box_outline_blank
OpenX
check_box_outline_blank
Parrable
check_box
PubMatic
check_box_outline_blank
RTB House
check_box
SharedID by Prebid.org
check_box_outline_blank
Tapjoy
check_box_outline_blank
Unified ID 2.0
check_box_outline_blank
Vungle

Reply:
I guess a link to the Google docs or the respective ad manager page would be a more reliable source. WDYT?